Week2/LaurAlejandraC/spring-core/src/main/java/com/globant/bootcamp/spring/springcore/data/CustomAccountRepository.java

```diff
@@ -4,10 +4,14 @@
 
 import com.globant.bootcamp.spring.springcore.domain.Account;
 
+import javax.annotation.Resource;
+
 @Repository
 public class CustomAccountRepository implements AccountRepository {
+    @Resource
```

Comment:
Please don't use Resource, let's use @Autowired annotation cuz resource is from javax.annotation package and remember that we are trying to use Spring's libraries :)